pkg/test/echo/docker/Dockerfile.app_sidecar_centos

```diff
@@ -0,0 +1,35 @@
+FROM centos:8
```

Comment:
For clarity - this image is not something we ship to users, just used as an example/testing